coffeecard/CoffeeCard.WebApi/Controllers/v2/AccountController.cs

```diff
@@ -31,6 +32,27 @@ public AccountController(IAccountService accountService, ClaimsUtilities claimsU
             _claimsUtilities = claimsUtilities;
         }
 
+                /// <summary>
+        /// Register a new account. A account is required to verify its email before logging in
+        /// </summary>
+        /// <param name="registerDto">Register data object</param>
+        /// <response code="201">Successful account creation</response>
+        /// <response code="409">Email already registered</response>
+        [HttpPost("register")]
+        [AllowAnonymous]
+        [ProducesResponseType(typeof(MessageResponseDto), StatusCodes.Status201Created)]
+        [ProducesResponseType(typeof(MessageResponseDto), StatusCodes.Status409Conflict)]
+        public async Task<ActionResult<MessageResponseDto>> Register([FromBody] RegisterDto registerDto)
+        {
+            await _accountService.RegisterAccountAsync(registerDto.Name, registerDto.Email, registerDto.Password, registerDto.ProgrammeId);
+
+            return Created("get", new MessageResponseDto()
```

Comment:
Have you verified that the `get` URI is the correct one? Since this is the V2 API, we're currently relying on the V1 for GET Account. 
Also, a would prefer a static reference to the URI if possible. Normally that can be achieved when actions are in the same Controller but not sure what is possible here, since you are relying on the GET from V1

Reply:
I think I hardcoded the path to the v1